concordium-sdk/src/main/java/com/concordium/sdk/ClientV2MapperExtensions.java

```diff
@@ -100,4 +128,332 @@ static BlockIdentifier to(FinalizedBlockInfo finalizedBlockInfo) {
                 .blockHeight(to(finalizedBlockInfo.getHeight()))
                 .build();
     }
+
+    static com.concordium.grpc.v2.AccountIdentifierInput to(AccountRequest accountIdentifier) {
+        var builder = AccountIdentifierInput.newBuilder();
+
+        switch (accountIdentifier.getType()) {
+            case ADDRESS:
+                builder.setAddress(to(accountIdentifier.getAddress()));
+                break;
+            case INDEX:
+                builder.setAccountIndex(to(accountIdentifier.getIndex()));
+                break;
+            case CREDENTIAL_REGISTRATION_ID:
+                builder.setCredId(to(accountIdentifier.getRegistrationId()));
+                break;
+        }
+
+        return builder.build();
+    }
+
+    static CredentialRegistrationId to(com.concordium.sdk.transactions.CredentialRegistrationId registrationId) {
+        return CredentialRegistrationId.newBuilder()
+                .setValue(to(registrationId.getRegId()))
+                .build();
+    }
+
+    static AccountIndex to(com.concordium.sdk.responses.AccountIndex index) {
+        return AccountIndex.newBuilder().setValue(index.getIndex().getValue()).build();
+    }
+
+    static AccountAddress to(com.concordium.sdk.transactions.AccountAddress address) {
+        return AccountAddress.newBuilder()
+                .setValue(to(address.getBytes()))
+                .build();
+    }
+
+    static com.concordium.sdk.responses.accountinfo.AccountInfo to(AccountInfo account) {
```

Comment:
I think this function should be a static member of `AccountInfo` and then called `from`.

Otherwise I think this file will get out of hand.

Reply:
And the same comment for similar functions present here.  Please move them out to the respective classes and make `from` functions there. I think that will be easier to manage in the end.

Reply:
@EmilLa1  - Yes the file is getting bigger. 
Can i suggest breaking this file into smaller files with non static functions. Then putting them in the ClientV2 Constructor. 
Wanted to keep the serialization / deserialization logic out of the respective classes. Simillar to what Object Mapper and other. Basically meaning that the Client Class dosent know of any Proto class.

Reply:
Can we create an issue for this and then fix it. 
Basically we want to create a non static class. And Client then depends on an object of this file.
This would allow for better testing since in bigger Objects link AccountInfo there are multiple branches which should be tested and its better that the mapping code is tested independently of the Client code.

Reply:
Yes. I think that is OK. :+1: